src/entrypoints/core.ts

```diff
@@ -99,3 +100,5 @@ window.addEventListener("error", (e) => {
     });
   }
 });
+
+window.addEventListener("haptic", (e) => handleHaptic(e.detail));
```

Comment:
We should not add it to `core.ts`, that file should be kept as lean as possible with only bootstrapping the things that are essential for fetching the data. Add it to the Home Assistant state mixins.